### ISSUE TITLE
Avatar resource name announcement issue screen reader

### ISSUE THREAD

**Issue by taauntik** [bryntum/support#11779]
[Forum post](https://forum.bryntum.com/viewtopic.php?f=51&t=33802&p=172405#p172405)

Hi Team,

We are facing focus & announcement issues of assigned resource names which are in avatar form.

Example:
https://bryntum.com/products/gantt/examples/collapsible-columns/

Windows + Chrome + NVDA
Individual avatars within the cell are navigable, but the screen reader announces only partial names (single words instead of full names). Screenshot attached.
When focus is on a cell containing multiple avatars, NVDA does not announce all resource names, unlike on macOS. 

![avatarNotAnnouncement.png](https://forum.bryntum.com/download/file.php?id=35987)
